app/scripts/metamask-controller.js

```diff
@@ -5486,7 +5486,7 @@ export default class MetamaskController extends EventEmitter {
       case KeyringType.lattice:
       case KeyringType.qr:
       case KeyringType.ledger:
-        return 'hardware';
+        return KEYRING_DEVICE_PROPERTY_MAP[keyringType];
```

Comment:
Nit: But the jsdoc is now wrong, since we're not returning `'hardware'` anymore!